src/renderer/components/settings-execution.tsx

```diff
@@ -125,7 +153,27 @@ export class ExecutionSettings extends React.Component<ExecutionSettingsProps, {
             />
           </FormGroup>
         </Callout>
+        <br />
+        <Callout>
+          <FormGroup>
+            <span style={{ marginRight: 4 }}>
+              You can change the default package manager to 'npm' or 'yarn'. The
```

Comment:
Something like this might make sense for the text to give more context on what the selected package manager does:

> Electron Fiddle will install packages on runtime if they are imported within your fiddle with `require`. It uses [npm](https://www.npmjs.com/) as its package manager by default, but [Yarn](https://classic.yarnpkg.com/lang/en/) is also available.